playbooks/collect_activity/stats.py

```diff
@@ -0,0 +1,66 @@
+#!/edx/bin/python.edxapp
+# pylint: skip-file
+
+from ConfigParser import ConfigParser
+import gzip
+import os
+import re
+import sys
+
+import django
+
+
+nginx_access_pattern = re.compile(r'- - (?P<ipaddress>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})')
+log_path = '/edx/var/log/nginx'
+
+
+if __name__ == '__main__':
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'lms.envs.aws')
+    os.environ.setdefault('SERVICE_VARIANT', 'lms')
+    django.setup()
+
+    unique_hits = set()
+    files = sorted([
```

Comment:
@bdero Rename this to `log_files`, perhaps, to be more specific about what these files are/contain.